File: Contest/Biweekly/Biweekly Contest 169/3736. Minimum Moves to Equal Array Elements III/Solution.java
class Solution {
    public int minMoves(int[] nums) {
        // Time Complexity : O(N)
        // Space Complexity : O(1)
        int maxNum = Arrays.stream(nums).max().getAsInt();
        int minMove = 0;
        for (int num: nums) {
            minMove += maxNum - num;
        }
        return minMove;
    }
}

File: Contest/Biweekly/Biweekly Contest 169/3736. Minimum Moves to Equal Array Elements III/Solution.py
from typing import List

class Solution:
    def minMoves(self, nums: List[int]) -> int:
        # Time Complexity : O(N)
        # Space Complexity : O(1)
        n = len(nums)
        maxNum = max(nums)
        minMove = 0
        for num in nums:
            minMove += maxNum - num
        return minMove